lib/core/boot/boot.ts

```diff
@@ -79,11 +84,38 @@ export const bootstrap = (): FluxifyServer => {
 						if (targetRoute.schema.body) body = targetRoute.schema.body.parse(body);
 					}
 
+					if (useCache) {
+						if (global.server.cache.length > config.cacheLimit) {
+							global.server.cache.pop();
+						}
+						global.server.cache = global.server.cache.filter((entry) => entry.exp > Date.now());
+						const hit = global.server.cache.find(
+							(entry) => entry.url === request.url && entry.jwt === (jwt as { id?: string })?.id,
+						);
+						if (hit) {
+							debug(`cache hit on route ${url.pathname}`);
+							return createResponse(hit.data, hit.status, time, {
```

Comment:
Shouldn't this be a 304 "Not Modified"?

Reply:
No because a client which has never requested that endpoint before would get absolutely no data at all.